fdp/validator.py

```diff
@@ -13,9 +21,13 @@ def _validate(data, shapes_file, fdp=False):
         s_set = set([s for s, p, o in g])
```

Comment:
`set([s for s, p, o in g])` could also be written `set(g.subjects()))` -- and is perhaps more expressive?